spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -94,9 +94,9 @@ function InitChannel(portId: string, connectionId: string) returns (nil){
 `RegisterInterchainAccount` is called on the `OnChanOpenTry` step during the channel creation handshake.
 
 ```typescript
-function RegisterInterchainAccount(accAddr: string, counterpartyPortId: string) returns (nil){
+function RegisterInterchainAccount(counterpartyPortId: string, connectionID: string) returns (nil) {
    // checks to make sure the account has not already been registered
-   // creates a new address on chain 
+   // creates a new address on chain deterministically given counterpartyPortId and underlying connectionID
```

Comment:
Ref: https://github.com/cosmos/ibc-go/issues/767

Reply:
Is it worth mentioning that it's the host side connection-id?